api/lib/infrastructure/datasources/airtable/skill-datasource.js

```diff
@@ -55,4 +55,9 @@ module.exports = datasource.extend({
     return _.filter(skills, { status: ACTIVATED_STATUS, competenceId });
   },
 
+  async findByName(name) {
+    const skills = await this.list();
+    return _.filter(skills, { name });
```

Comment:
il peut y avoir plusieurs skills avec le meme nom?

Reply:
Sinon ca serait plutot 
```suggestion
    return _.find(skills, { name });
```

Reply:
@aceol c.f. réponse apportée sous le commentaire identique d'yvo